### ISSUE TITLE
Failed to install lumo because of `repo1.maven.org: Name or service not known`

### ISSUE THREAD

**Issue by leira** [NixOS/nixpkgs#79781]
I had lumo installed in my Nix on Debian setup. Today, I try to update my system, I met this build error for lumo: `java.net.UnknownHostException: repo1.maven.org: Name or service not known` . The failure log is here: https://pastebin.com/wWQrhY7Y.

I tried to ping repo1.maven.org and could reach it. I wonder if it is a Nix sandbox thing?

```
Caused by: java.net.UnknownHostException: repo1.maven.org: Name or service not known
        at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
        at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:929)
        at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1515)
        at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
        at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1505)
        at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1364)
        at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1298)
        at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45)
        at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:263)
        at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:162)
        at 
...
```

I'm on Nix unstable, with nixpkgs version "20.03pre211998.57ed5db343f":

```
$ nix-instantiate --eval -E '(import <nixpkgs> {}).lib.version'
"20.03pre211998.57ed5db343f"
```

**Comment by veprbl** [comment]
cc @hlolli

**Comment by hlolli** [comment]
I was looking into this earlier tonight, I was already notified about this from @Mic92 

I discussed with core clojure developers (including Alex Miller) and it seems that there was a regression introduced where a network call is made without any clear reason. I will find a solution tomorrow night via https://github.com/NixOS/nixpkgs/pull/59414 it's always possible to start and run clojure trough java directly, or preferably use older version of clojure command line tool/exec while I make sure this regression gets looked at. I hope this isn't blocking too many people.

**Comment by leira** [comment]
Thanks @hlolli, looking forward to the fix.

I was on Nix on Debian, how do I help to test #59414? I suppose I should add your remote, and checkout hlolli:lumo-bump. What's next then?

**Comment by raboof** [comment]
I think this can be closed since #59414 is merged?
